python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1171,6 +1172,100 @@ def _get_abs_layer_name(node):
                                                      params, num_layers)
         return sym
 
+_control_flow_nodes = ['Merge', 'Switch', 'NextIteration', 'Exit', 'Enter', 'LoopCond']
+
+class Branch:
+    """A class contains the components that are used to build up a Relay if
+    node.
+    """
+    def __init__(self):
+        self._if = None
+        self.cond_vars = set()
+        self.cond = None
+        self.true_branch = None
+        self.false_branch = None
+
+    def _if_node(self):
+        from tvm import relay
+
+        cond_vars = []
+        bind_map = {}
+        for i, var in enumerate(list(self.cond_vars)):
+            if not isinstance(var, _expr.Var):
+                raise TypeError("var is expected to be _expr.Var type, but "
+                                "received {}".format(repr(var)))
+            v = relay.var("cond_var" + str(i),
+                          type_annotation=var.type_annotation)
+            cond_vars.append(v)
+            bind_map[var] = v
+
+        self.cond = relay.bind(self.cond, bind_map)
+        cond = relay.op.min(self.cond)
```

Comment:
just trying to understand why is `op.min` required here?

Reply:
This is a little tricky. `cond` will return a tensor that contains `True` and `False`. `Min` checks if there is any `false`. If so, this condition doesn't not hold.

Reply:
mind add a comment here? as it might confuse ppl.

Reply:
sure